tests/destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4,9 +4,56 @@ exports[`destructuring.js 1`] = `
 const [one, two = null, three = null] = arr;
 a = ([s=1,]) => 1
 const { children, ...props } = this.props
+
+const { user: { firstName, lastName } } = this.props;
+
+const {
+  name: { first, last },
+  organisation: { address: { street: orgStreetAddress, postcode: orgPostcode } }
+} = user;
+
+function f({ data: { name } }) {}
+
+const UserComponent = function({
+  name: { first, last },
+  organisation: { address: { street: orgStreetAddress, postcode: orgPostcode } },
+}) {
+  return
+};
+
+const { a, b, c, d: { e } } = someObject;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 const [one, two = null, three = null] = arr;
 a = ([s = 1]) => 1;
 const { children, ...props } = this.props;
 
+const {
+  user: { firstName, lastName }
+} = this.props;
```

Comment:
I'm on the fence with this one... I personally prefer it collapsed, but don't care either way